__tests__/news.test.js

```diff
@@ -0,0 +1,30 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => seed(testData));
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET", () => {
+  it("/api/topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        const { topics } = response.body;
+
+        expect(Array.isArray(topics)).toBe(true);
+        expect(topics.length === 3).toBe(true);
+
+        topics.forEach((topic) => {
+          expect(topic).toHaveProperty("slug", expect.any(String));
+          expect(topic).toHaveProperty("description", expect.any(String));
+        });
+      });
+  });
+});
```

Comment:
This looks great so far!

Id like to see some error handling however at this stage for requesting an invalid endpoint. Say if someone requested '/api/topicz'